src/components/EventDetails/EventDetails.js

```diff
@@ -907,27 +906,31 @@ export class EventDetail extends React.Component<Props> {
         };
         break;
       case PAYMENT_NETWORK_ACCOUNT_WITHDRAWAL:
-        eventData = {
-          buttons: [
-            {
-              title: t('button.withdrawMore'),
-              onPress: this.PPNWithdraw,
-              secondary: true,
-            },
-          ],
-        };
+        if (isArchanovaAccountActive) {
```

Comment:
Similarly, consider inverting the if for readability here and in next occurrences.